collector/external-dns/README.md

```diff
@@ -0,0 +1,112 @@
+---
+# Ingest metrics using the External DNS integration
+
+The OTel Collector has a variety of [third party receivers](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/master/receiver) that provide integration with a wide variety of metric sources.
+
+Please note that not all metrics receivers available for the OpenTelemetry Collector have been tested by Lightstep Observability, and there may be bugs or unexpected issues in using these contributed receivers with Lightstep Observability metrics. File any issues with the appropriate OpenTelemetry community.
+{: .callout}
+
+## Requirements
+
+* OpenTelemetry Collector Contrib v0.51.0+
+
+## External DNS supports
+
+ExternalDNS allows to configure external services, like [AWS Route53](https://github.com/kubernetes-sigs/external-dns/blob/master/docs/tutorials/aws.md), [Cloudflare DNS](https://github.com/kubernetes-sigs/external-dns/blob/master/docs/tutorials/cloudflare.md) and other ones. However for this example we provide configuration for CoreDNS.
+
+## Prerequisites for local installation with CoreDNS
+
+You must have a Lightstep Observability [access token](/docs/create-and-manage-access-tokens) for the project to report metrics to.
+
+#### minikube
+
+You can use any approach to managing your cluster, but the Makefile builds a cluster in `minikube`.
+
+#### helm
+
+We use helm charts to install some apps in this example.
+
+## Running the Example
+
+You can run this example with `make all` in this directory.
+After tests it's just need to call `make delete-cluster`.
+
+## Steps
+
+### 1. Create a cluster
+
+First you'll need to create a cluster by a method of your choice. `minikube start` works well on Linux.
+
+### 2. Required libraries
+
+External DNS requires ETCD, DNS manager, for this example we picked CoreDNS. We add repos for coreDNS, otel-collector.
+
+### 3. Installation
+
+#### a. Install etcd
+
+etcd is used to manage etcd clusters, will be required for CoreDNS.
```

Comment:
I don't think this should be "etcd clusters". I'm guessing this was meant to say "coreDNS clusters," but I leave it to you.